arch/arm64/src/rk3399/hardware/mx8mp_pinmux.h

```diff
@@ -0,0 +1,197 @@
+#define IOMUXC_GRF_GPIO2C_IO07
+#define IOMUXC_GRF_GPIO2D_IO07
+
+#define IOMUXC_GRF_GPIO3A_IO01_GPIO                         GRF_GPIO3A_IOMUX_ADDR, 0b00, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO3A_IO01_DP_HOTPLUG                   GRF_GPIO3A_IOMUX_ADDR, 0b01, 0x00000000, 0x0, 0x00000000
+
+#define IOMUXC_GRF_GPIO3B_IO02_GPIO
+#define IOMUXC_GRF_GPIO3B_IO02_MAC_RXER
+#define IOMUXC_GRF_GPIO3B_IO02_I2C5TRACKPAD_SDA
+
+
+#define IOMUXC_GRF_GPIO3B_IO03_GPIO
+#define IOMUXC_GRF_GPIO3B_IO03_MAC_CLK
+#define IOMUXC_GRF_GPIO3B_IO03_I2C5TRACKPAD_SCL
+
+#define IOMUXC_GRF_GPIO4A_IO00_GPIO                         GRF_GPIO4A_IOMUX_ADDR, 0b00, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4A_IO00_I2S_CLK                      GRF_GPIO4A_IOMUX_ADDR, 0b01, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4A_IO00_JTAG_TRACE_CTL               GRF_GPIO4A_IOMUX_ADDR, 0b10, 0x00000000, 0x0, 0x00000000
+
+//todo add offset for pins other than [0]
+// GPIO 4C
+#define IOMUXC_GRF_GPIO4C_IO00_GPIO                         GRF_GPIO4C_IOMUX_ADDR, 0x0, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO00_UART2DBGD_SIN                GRF_GPIO4C_IOMUX_ADDR, 0x1, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO00_I2C3HDMI_SDA                GRF_GPIO4C_IOMUX_ADDR, 0x2, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO00_HDMII2C_SDA                 GRF_GPIO4C_IOMUX_ADDR, 0x3, 0x00000000, 0x0, 0x00000000
+
+#define IOMUXC_GRF_GPIO4C_IO01_GPIO                         GRF_GPIO4C_IOMUX_ADDR, 0x0, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO01_UART2DBGD_SOUT                GRF_GPIO4C_IOMUX_ADDR, 0x1, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO01_I2C3HDMI_SCL                GRF_GPIO4C_IOMUX_ADDR, 0x2, 0x00000000, 0x0, 0x00000000
+#define IOMUXC_GRF_GPIO4C_IO01_HDMII2C_SCL                 GRF_GPIO4C_IOMUX_ADDR, 0x3, 0x00000000, 0x0, 0x00000000
+
+#define IOMUXC_GRF_GPIO4C_IO02_GPIO                 
+#define IOMUXC_GRF_GPIO4C_IO02_PWM_0                
+#define IOMUXC_GRF_GPIO4C_IO02_VOP0_PWM             
+#define IOMUXC_GRF_GPIO4C_IO02_VOP1_PWM             
+
+#define IOMUXC_GRF_GPIO4C_IO03_GPIO                   
+#define IOMUXC_GRF_GPIO4C_IO03_UART2DBGC_SIN                  
+#define IOMUXC_GRF_GPIO4C_IO03_UARTHDCP_SIN            
+
+#define IOMUXC_GRF_GPIO4C_IO04_GPIO                   
+#define IOMUXC_GRF_GPIO4C_IO04_UART2DBGC_SOUT                  
+#define IOMUXC_GRF_GPIO4C_IO04_UARTHDCP_SOUT  
+
+#define IOMUXC_GRF_GPIO4C_IO05_GPIO                   
+#define IOMUXC_GRF_GPIO4C_IO05_SPDIF_TX       
+
+#define IOMUXC_GRF_GPIO4C_IO06_GPIO                   
+#define IOMUXC_GRF_GPIO4C_IO06_PWM_1  
+
+#define IOMUXC_GRF_GPIO4C_IO07_GPIO                   
+#define IOMUXC_GRF_GPIO4C_IO07_HDMI_CECINOUT
+#define IOMUXC_GRF_GPIO4C_IO07_EDP_HOTPLUG
+// GPIO 4D
+
+#define IOMUXC_GRF_GPIO4D_IO00_GPIO          GRF_GPIO4D_IOMUX_ADDR, HIWORD_UPDATE(0b0000,0b1111,0), 0x00000000, 0x0, 0x00000000
```

Comment:
Need to rename hiword update but this is how it would look like to specify a specific grf usage / config

Reply:
Yep sure